srcpkgs/i3lockr/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'i3lockr'
+pkgname=i3lockr
+version=0.1.0
+revision=1
+build_style=cargo
+hostmakedepends="python3"
+makedepends="xcb-util-image-devel"
+short_desc="Distort a screenshot and run i3lock"
+maintainer="Sinésio Neto <sinetoami@gmail.com>"
+license="MIT, Apache-2.0"
+homepage="https://github.com/owenthewizard/i3lockr"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=fc715687e21fd36f8927f45a5a6fbc04bf81e2e7bcbfa9866a511400551af314
+
+post_install() {
+	vlicense LICENSE-MIT.md
+	vlicense LICENSE-APACHE.md
```

Comment:
don't need to vlicense APACHE, only MIT

Reply:
In the `license` variable, Do I have to take it off too?